detection-rules/impersonation_netflix.yml

```
@@ -0,0 +1,19 @@
+name: "Brand impersonation: Netflix"
+description: |
+  Impersonation of Netflix.
+references:
+  - "https://news.trendmicro.com/2023/01/18/netflix-scams-2023-job-text-email/"
+type: "rule"
+severity: "low"
```

```suggestion
severity: "low"
authors:
  - twitter: "handle here"
    name: "min0k"```
    
 Also feel free to credit yourself, you can use your twitter handle or just your name or none at all! Just want to make sure you get credit! Really appreciate the contribution!!

---

Feel free to resolve this without committing and add authors manually: [here](https://github.com/search?q=repo%3Asublime-security%2Fsublime-rules%20author&type=code) are some examples.